htdocs/holiday/class/holiday.class.php

```diff
@@ -1630,6 +1630,7 @@ public function fetchUsers($stringlist = true, $type = true, $filters = '')
 					$sql .= " WHERE u.entity IN (".getEntity('user').")";
 				}
 				$sql .= " AND u.statut > 0";
+				$sql .= " AND u.employee = 1"; // We only want employee users for holidays
```

Comment:
Why not the boss can't also create holiday to have its record validated and visible (even if I agree he validates himself)

Reply:
This fix is mostly about the holiday counters. The boss has no counter or if so, he must be considered as an employee. This doesn't prevent the boss to record it's holidays and have it shared on the company planning.